Improve coreapi importing

gsvr opened the pull request:
## Description
Closes #5918 
Import coreapi from rest_framework.compat, not directly.

DEKHTIARJonathan commented:
Please release a new version on PyPI as soon as this PR is merged ;)

Thanks for the work

gsvr commented:
@carltongibson I hope I opened this PR in the right way, first time contributing :sweat_smile: 
I did two additional things to what you stated in the issue:
1. Added `and coreschema is not None`, not sure if this was necessary but it looked safer.
2. Changed the import in `test_renderers.py` - I doubt this is necessary but it's more consistent with other imports in tests.

carltongibson commented:
@gsvr Thanks for this. 

This will go into a 3.8.2. There are a couple of issues for that, so it'll be first half of next week. If you cannot wait until then you should [install directly from GitHub](https://pip.pypa.io/en/stable/reference/pip_install/#git). (Pip has first-class support for this. It's not something to avoid.)

gsvr commented:
@carltongibson It's not a problem for me, I just pinned my version to `<3.8` for now.  Thanks for the quick responses :)

carltongibson commented:
v3.8.2 is now available on PyPI. Thanks all.

LuisHCK commented:
But nothing is changed
![screenshot from 2018-10-09 11-46-22](https://user-images.githubusercontent.com/5175137/46687806-00f14400-cbb9-11e8-8e22-bdde85f76de5.png)
Yes, I have already installed the latest version.
Yes, I already installed directly from the repository

xordoquy commented:
@LuisHCK please, don't use passive aggressive tone.
We - anyone that contributed here - have no part in your code not working as you'd have wished/expected.
You just don't bring anything valuable to the issue here beside negative comments.
Please, be constructive. Write the simplest possible test case where you can show your issue and open a PR so we may look at it.

rpkilby commented:
@LuisHCK. These changes are included in the 3.8.2 release (see: [here](https://github.com/encode/django-rest-framework/blob/3.8.2/rest_framework/authtoken/views.py)). I would check again what version of DRF is installed. You may also want to recreate your virtual environment.
